specs/bridges/liquity/TroveBridge/readme.md

```diff
@@ -0,0 +1,92 @@
+# Spec for Trove Bridge
+
+## What does the bridge do? Why build it?
+The bridge allows user to borrow LUSD by depositing ETH as collateral.
+
+## What protocol(s) does the bridge interact with ?
+This bridge interacts with Liquity and Uniswap V3 protocols.
+Uniswap is used when user wants to repay debt with collateral (ETH is "flash swapped" to LUSD, debt gets repaid,
+collateral withdrawn, flash swap gets paid for with withdrawn collateral).
+
+## What is the flow of the bridge?
+There are 3 flows supported by the bridge: borrowing, repaying and redeeming.
+
+> Note: Before the bridge gets used someone has to open the bridge's Trove by calling bridge's `openTrove` function.
+
+### Borrowing
+Borrowing flow is simple.
+User provides ETH on the input and gets LUSD on the ouput (+ accounting token used to rerpresent user's collateral and
+debt).
+Since the collateral ratio is fixed by the bridge, user gets the amount of LUSD out based only on the amount
+of collateral provided and the collateral ratio.
+The borrowing interaction would revert in case the bridge's Trove is not active.
+
+> Note: Fixed collateral ratio in this case means that no user can ever move the collateral ratio of the bridge's Trove.
```

Comment:
Might need to add a comment that the value will move with market movements, just not be influenced by the individual user?

Reply:
Addressed in https://github.com/AztecProtocol/aztec-connect-bridges/pull/298/commits/7aa015296f0dcacf8b7e148b87f33a848ed880a3